client/src/client.rs

```diff
@@ -353,6 +353,22 @@ pub trait RpcApi: Sized {
         self.call("getblockheader", &[into_json(hash)?, true.into()]).await
     }
 
+    /// Compute per block statistics for a given window. All amounts are in satoshis.
+    /// https://developer.bitcoin.org/reference/rpc/getblockstats.html
+    async fn get_block_stats(
+        &self,
+        block_hash: Option<&bitcoin::BlockHash>,
```

Comment:
might make more sense to do param hash_or_height: HashOrHeight and define a custom enum to get rid of the panic branch.